ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/index.tsx

```diff
@@ -46,6 +49,7 @@ const RunComponent = () => {
         repositoryId: Number.parseInt(params.repoId),
         ortRunIndex: Number.parseInt(params.runIndex),
       }),
+    refetchInterval: pollInterval,
```

Comment:
It would probably make sense to disable refetch after the run is finished, as it should not change after that.

Reply:
Polling / stop polling has been now tested to work with FINISHED and ACTIVE runs, respectively.